scripts/ska3_builder

```diff
@@ -10,20 +10,20 @@ if os.uname().sysname == "Darwin":
 
 parser = argparse.ArgumentParser(description="Build Ska Conda packages.")
 
-parser.add_argument("mode", type=str, help="The build mode. Either 'all' for all "
-                                           "packages, 'updated' for updated "
-                                           "packages, or a specific package name.")
-parser.add_argument("--build_root", default=".", type=str, 
+parser.add_argument("package", type=str, nargs="?",
+                    help="Package to build.  All we be built as needed if no package supplied")
+parser.add_argument("--tag", type=str,
+                    help="Optional tag, branch, commit for git checkout.  Only applies to single package build")
```

Comment:
`Optional tag, branch, or commit to build for single package build (default is tag with most recent commit)`